backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/allocation/transfer/HUTransformService.java

```diff
@@ -973,6 +979,13 @@ public List<I_M_HU> tuToNewLUs(
 		}
 	}
 
+	private void copyAttributesToNewLU(final I_M_HU newLuHU, final I_M_HU sourceTuHU)
```

Comment:
i see `copyAttributesToNewLU` is copying only LotNumber from a TU to a new LU.
I guess u thought there will  be more attributes in future...
... but until the future comes, i suggest to rename it `copyLotNumberFromTuToNewLu`,
or simply inline it, as u wish.

Reply:
inlined.